llvm/test/Transforms/LoopVectorize/AArch64/low_trip_memcheck_cost.ll

```diff
@@ -0,0 +1,187 @@
+; REQUIRES: asserts
+; RUN: opt -p loop-vectorize -debug-only=loop-vectorize -S -disable-output < %s 2>&1 | FileCheck %s
+
+target triple = "aarch64-unknown-linux-gnu"
+
+
+define void @outer_no_tc(ptr nocapture noundef %a, ptr nocapture noundef readonly %b, i64 noundef %m, i64 noundef %n) {
```

Comment:
It might be worth adding a version with no outer loop as a baseline. And maybe a comment about what the test is testing.